LoopKitUI/Views/Settings Editors/DeliveryLimitsEditor.swift

```diff
@@ -28,26 +28,33 @@ public struct DeliveryLimitsEditor: View {
     @Environment(\.dismiss) var dismiss
     @Environment(\.authenticate) var authenticate
 
+    private let lowestCarbRatio: Double?
+
     public init(
         value: DeliveryLimits,
         supportedBasalRates: [Double],
         scheduledBasalRange: ClosedRange<Double>?,
         supportedBolusVolumes: [Double],
+        lowestCarbRatio: Double?,
         onSave save: @escaping (_ deliveryLimits: DeliveryLimits) -> Void,
         mode: PresentationMode = .settings
     ) {
         self._value = State(initialValue: value)
         self.initialValue = value
         self.supportedBasalRates = supportedBasalRates
+        let basalGuardrail = Guardrail.maximumBasalRate(supportedBasalRates: supportedBasalRates, scheduledBasalRange: scheduledBasalRange, lowestCarbRatio: lowestCarbRatio)
         if let maximumScheduledBasalRate = scheduledBasalRange?.upperBound {
-            self.selectableBasalRates = Array(supportedBasalRates.drop(while: { $0 < maximumScheduledBasalRate }))
+            selectableBasalRates = Array(supportedBasalRates.drop(while: { $0 < maximumScheduledBasalRate }))
```

Comment:
[nit] `self.` prefix to be consistent with the rest of this initializer?

Reply:
weird, don't know where that went.  Will fix 👍